### PR TITLE
feat: use pingcap base image

### PR DESCRIPTION
**Pull request by lijie0123**
### What problem does this PR solve?
The current base image lack matainence and may contains security problem.

### What is changed and how does it work?
Use pingcap unify base image maintained by EE and security team

### Code changes

- [x] Has Go code change
- [ ] Has CI related scripts change

### Tests

- [ ] Unit test 
- [ ] E2E test 
- [x] Manual test 
- [ ] No code 

### Side effects

- [ ] Breaking backward compatibility 
- [ ] Other side effects: 

### Related changes

- [ ] Need to cherry-pick to the release branch 
- [ ] Need to update the documentation 

### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note

```

### CODE REVIEW

File: images/br-federation-manager/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,4 @@
-FROM alpine:3.14
+FROM pingcap/tools-base:v1
```
Comment (csuzhangxc) [inline images/br-federation-manager/Dockerfile:1]: what's the difference between `pingcap-base` and `tools-base`?

Comment (csuzhangxc) [inline images/br-federation-manager/Dockerfile:1]: there is `Dockerfile.e2e` for br-federation-manager too.

Comment (lijie0123) [reply images/br-federation-manager/Dockerfile:1]: tools-base contains bind-tools, and wget, which some tools usually need

Comment (lijie0123) [reply images/br-federation-manager/Dockerfile]: added

Comment (lijie0123) [reply images/br-federation-manager/Dockerfile]: change to single pingcap-base

**Review by WangLe1321 — APPROVED** [review]

**Pull request merged**